distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Parse_Values_Helper.enso

```diff
@@ -1,5 +1,19 @@
 from Standard.Base import all
 
+import project.Value_Type.Value_Type
+
 ## PRIVATE
 valid_parse_targets =
     ['Integer', 'Float', 'Date', 'Date_Time', 'Time', 'Boolean']
+
+## PRIVATE
+   Checks if the type is a valid argument for `parse`.
+is_a_valid_parse_target target_type:Value_Type -> Boolean =
+    case target_type of
+        Value_Type.Integer _ -> True
+        Value_Type.Float _ -> True
+        Value_Type.Date -> True
+        Value_Type.Date_Time _ -> True
+        Value_Type.Time -> True
+        Value_Type.Boolean -> True
+        _ -> False
```

Comment:
Much better than the old one 👍

Reply:
Ah, but now I realize why I wrote the old one the way it was written.

It actually made sense. It was written that way (checking constructor name) to guarantee consistency between `is_a_valid_parse_target` and the parse target selector widget. Now we actually lose that...

(I realized as I wanted to suggest to remove the `valid_parse_targets` list as it seemed obsolete, but it actually isn't since it is used in the widget)

Reply:
Would be good to have some level of ensuring consistency - e.g. let's at least keep the list of types and case-of in the same file (as is right now) - so if any change is done in one place, it will be easier to not-forget about the other.